### PR TITLE
Storyblok story analysis scripts

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Add scrips for dumping and analyzing CMS stories

For now I need them to deprecate and remove some of page flags, but could be useful in the future as well

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
* **#4114** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4114?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4113** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4113?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4112** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4112?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alebedev and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/scripts/storyblok-analyze-stories.cjs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// yarn node ./scripts/storyblok-analyze-stories.cjs < ./stories.json
+const main = async () => {
+  const stories = await readStoriesFromStdin()
+  for (const story of stories) {
+    const { hideMenu, hideFooter } = story.content
+    if (hideMenu || hideFooter) {
+      console.log(story.full_slug, story.content.component, { hideMenu, hideFooter })
```
Comment (gustaveen) [inline apps/store/scripts/storyblok-analyze-stories.cjs:7]: So this script just checks if hideMenu or hideFooter is set to true?

**Review by gustaveen — APPROVED** [review]
Looks good, should we maybe name them more specific to what the scripts do?

File: apps/store/scripts/storyblok-analyze-stories.cjs (hunk 1 above)
Comment (alebedev) [reply apps/store/scripts/storyblok-analyze-stories.cjs:7]: For now.  I'll update it further as I go through other flags to be deprecated

**Pull request merged**